tardis/io/tests/test_HDFWriter.py

```diff
@@ -123,3 +123,16 @@ def test_objects_write(tmpdir, attr):
         fname, key='/test/mock_class/nested_object/property')
     assert_array_almost_equal(
         actual.nested_object.property, nested_property)
+
+
+def test_snake_case():
+    obj = MockHDF(None)
```

Comment:
you don't need this line because `convert_to_snake_case` is a static method. You can simply write `MockHDF.convert_to_snake_case(...)`